core-bundle/src/Migration/Version408/Version480Update.php

```diff
@@ -348,26 +334,21 @@ public function runImportantPart(): void
                     $file['path']
                 );
             } else {
-                $updateData[':x'] = min(1, $file['importantPartX'] / $imageSize[0]);
-                $updateData[':y'] = min(1, $file['importantPartY'] / $imageSize[1]);
-                $updateData[':width'] = min(1 - $updateData[':x'], $file['importantPartWidth'] / $imageSize[0]);
-                $updateData[':height'] = min(1 - $updateData[':y'], $file['importantPartHeight'] / $imageSize[1]);
+                $updateData['x'] = min(1, $file['importantPartX'] / $imageSize[0]);
+                $updateData['y'] = min(1, $file['importantPartY'] / $imageSize[1]);
+                $updateData['width'] = min(1 - $updateData['x'], $file['importantPartWidth'] / $imageSize[0]);
+                $updateData['height'] = min(1 - $updateData['y'], $file['importantPartHeight'] / $imageSize[1]);
             }
 
-            $this->connection
-                ->prepare('
-                    UPDATE
-                        tl_files
-                    SET
-                        importantPartX = :x,
-                        importantPartY = :y,
-                        importantPartWidth = :width,
-                        importantPartHeight = :height
-                    WHERE
-                        id = :id
-                ')
-                ->executeStatement($updateData)
-            ;
+            $this->connection->executeStatement('
+                UPDATE
```

Comment:
```suggestion
            $this->connection->executeStatement(
                'UPDATE
```
Seems to be inconsistent? _(just a nitpick, feel free to ignore)_

Reply:
Fixed in 1068a12064e06624a77d7660b92bfa3ec5bec387.